src/model/dataTypes/ComplexDataType.ts

```diff
@@ -2,14 +2,47 @@ import { normalizeTypeName } from "../../utils";
 import IDataType from "./IDataType";
 import { primitiveTypeCheck } from "./primitiveDataTypes";
 
+// This cache is a map of namespace strings, whose objects are a map of types under that namespace
+// E.g.: { "FHIR": { "Account": { /* Account obj }, "Activity": { /* Activity Obj */ } }
+type CachedNamespace = {
+  [key: string]: ComplexDataType;
+};
+type CacheType = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use an interface instead of a type literal.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5648116)